packages/scoutgame/src/points/refreshBuilderLevels.ts

```diff
@@ -8,6 +8,8 @@ import { calculateBuilderLevels } from './calculateBuilderLevel';
 export async function refreshBuilderLevels({ season = getCurrentSeasonStart() }: { season?: ISOWeek }) {
   const levels = await calculateBuilderLevels({ season });
 
+  log.info(`Refreshing builder levels for season ${season} with ${levels.length} builders`);
```

Comment:
this is a side effect so log.debug would be better

Reply:
I grouped these into a single log.info